editor/static/scripts/editor.js

```diff
@@ -68,7 +68,7 @@ function register(layout) {
             editor.container.style.background = "white";
             editor.focus();
 
-            saveTimer = setInterval(save, 5000);
+            saveTimer = setInterval(() => save(), 5000);
```

Comment:
What does this change do?

Reply:
save has an optional argument which I don't want to provide here. But setInterval will pass the timer object in as the first parameter unless I write it like this.

This was related to the same issue, so I decided to fix it in the same PR.